### ISSUE TITLE
Is it possible to modify the look of the exchange 

### ISSUE THREAD

**Issue by Jacob807** [3s3s/opentrade#137]
If so what steps would need to be taken to do so. 
Thanks

**Comment by jonn4y** [comment]
how do you mean?
the layout? some other developers have changed the layout of their exchanges like these:
http://cointrades.pw
https://orionbtc.com/market/BTC-XMCC

or do you mean change the colours?

**Comment by Jacob807** [comment]
Yeah I mean by the layout like making the charts look. Bigger on the page and bit more interactive. Things of that nature is what I was asking.

**Comment by hashbook** [comment]
i made a pull request of my layout, charts and other updates (buy / sell form calculation - by default they are wrong. original developer has math problems) but it was rejected saying that these are not the updates that will increase performance. I can give you invite to private repo where all these updates are stored.

**Comment by daygle** [comment]
@hashbook may I see your private repo please?

I'm curious about your mention about math problems.

**Comment by CryptorClub** [comment]
https://greencoin.online Switch toggler at top right corner for nightmode...
Like my comment, if you ready for pull request.

**Comment by jonn4y** [comment]
Nice work
